modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -148,7 +148,7 @@ RETURN p
 
 Description of the returned code::
 Using shortest path with an exhaustive search fallback might cause query slow down since shortest path graph algorithms might not work for this use case.
-It is recommended to introduce a `WITH` to separate the `MATCH` containing the shortest path from the existential predicates on that path.
+It may be possible to introduce a `WITH` to separate the `MATCH` containing the shortest path from the existential predicates on that path.
```

Comment:
```suggestion
It may be possible to introduce a `WITH` to separate the `MATCH` containing the shortest path from the existential predicates on that path and thereby eliminating the need for the fallback.
```